query/lambda/excessive_timout.sql

```diff
@@ -0,0 +1,27 @@
+with lambda_duration as (
+  select 
+    name,
+    avg(average:: numeric) as avg_duration
+  from 
+    aws_lambda_function_metric_duration_daily
+  where
+    date_part('day', now() - timestamp) <=7
+  group by
+    name
+)
+select
+  arn as resource,
+  case
+    when avg_duration is null then 'error'
+    when ((timeout :: numeric*1000)-avg_duration)/(timeout :: numeric*1000) > 0.1 then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when avg_duration is null then 'CloudWatch metrics not available for ' || title || '.'
```

Comment:
The reason can be confusing if we don't mention the lambda in it 
'CloudWatch metrics not available for ' > CloudWatch Lambda function metrics not available for ...

Reply:
Done